### ISSUE TITLE
Questions about the code.

### ISSUE THREAD

**Issue by BrightHaozi** [yxdyc/pFedGate#2]
Hello! We are interested in your work and thank you for opening the source code. We want to compare your method with ours in our paper. But we ran into some problems running the code. We conduct experiments on cifar-100 under dirichlet non-IID with alpha=0.5. However, the accuracy is very low and almost does not change during training. We also test on pathological noniid with 40 clients. The accuracy has been around 50% (some clients 50%, some clients 0%). We think this is due to the hyperparameter settings. Could you please give us some suggestions? Note that we adopt the hyperparameters specified in your paper. But there are still some hyperparameters we do not know how to set and we just adopt the default value in the code. We would really appreciate your help!

**Comment by yxdyc** [comment]
Hello, you can try to set  `lr_scheduler=multi_step` and `--seperate_trans=1` . If the optimization is still struggling, a bit of further searching for the hyper-parameters `lr_gating`, `lr_model` should be helpful, as the data after FL partition are different (albeit the same centralized dataset).  
We provide some HPO examples using wandb in the `scripts/sweep_hpo` directory, and some searched hyper-parameters that generally achieve good performance in [here](https://github.com/yxdyc/pFedGate/blob/main/scripts/README.md), where we have some discussion about the optimization. We hope this helps and thank you very much for your interest!

**Comment by BrightHaozi** [comment]
Thanks for your suggestions! We find we ignored the `--block_wise_prune` before. It is set to 0 in the code (default). When we set it to 1, the code works fine. Thanks again for releasing the code!
